app/server/utils/transfer_limits.py

```diff
@@ -205,36 +205,36 @@ def __init__(self,
 
```

Comment:
Add a check to raise error if total_amount and transfer_balance_fraction is applied

i.e. `if transfer_balance_fraction and total_amount: raise TransferLimitException`